src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -642,6 +656,65 @@ bool AreaTrigger::HasSplines() const
     return bool(_spline);
 }
 
+AreaTriggerCircularMovementInfo AreaTrigger::GetAreaTriggerCircularMovementInfo() const
+{
+    AreaTriggerCircularMovementInfo areaTriggerCircularMovementInfo;
+    AreaTriggerMiscTemplate const* areaTriggerMiscTemplate = GetMiscTemplate();
+
+    if (!areaTriggerMiscTemplate)
```

Comment:
how is this even possible?

Reply:
Possible on Ashamane because of DB spawned AreaTriggers. I can remove the check for now on TC if you prefer